src/labs/labs.css

```diff
@@ -410,11 +410,10 @@ tt {
 pre code {
   margin: 0;
   padding: 0;
-  white-space: pre;
   border: none;
   background: transparent;
   overflow-x: auto;
-  white-space: pre-wrap;
+  white-space: pre-wrap !important;
```

Comment:
Could you add a comment explaining why `!important` is needed here?

Reply:
@zigarn do you need help with this? If you explain why you needed to put this in I can take care of writing the comment if you want.

Reply:
Sorry, didn't have time to look back at it.
I would say that it's the only way I found to make it work for line wrapping.

There is a `code[class*="language-"], pre[class*="language-"]` with `white-space: pre` that is at the beginning of the `labs.css` (from reveal I guess) and applies to the element. Didn't found any other way to override it.

Reply:
OK, finally found a better way (thought it didn't work last time)